CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.1.14-dev
```

Comment:
we can go ahead and publish right away I think, if you want to remove the -dev (no other changes planned for this package)

Reply:
Done.

Reply:
You got the pubspec but not the changelog ;)

Reply:
Done, thanks :)